zebra-state/src/service/finalized_state/zebra_db/transparent.rs

```diff
@@ -217,6 +259,14 @@ impl DiskWriteBatch {
                     address_unspent_output,
                     (),
                 );
+
+                // Create a link from the AddressLocation to the new TransactionLocation in the database.
+                // Unlike the OutputLocation link, this will never be deleted.
+                let address_transaction = AddressTransaction::new(
+                    receiving_address_location,
+                    new_output_location.transaction_location(),
+                );
+                self.zs_insert(&tx_loc_by_transparent_addr_loc, address_transaction, ());
```

Comment:
Question: Do we also need to index transactions that sent to an address?

Reply:
You're right, thanks for catching that. Done in bcd8d0b8c